### DISCUSSION TITLE
Do 3.4.0 English models use floret vectors?

### DISCUSSION THREAD

**Discussion by kinghuang** [explosion/spaCy#11541] [Help: Model Advice] [answered]
The [release notes for spaCy 3.4.0](https://spacy.io/usage/v3-4) mention that the [English pipelines](https://spacy.io/models/en) have new word vectors, and link to [spacy-vectors-builder](https://github.com/explosion/spacy-vectors-builder/tree/lang/en). Does this mean that `en_core_web_md` and `en_core_web_lg` now have floret vectors? Or, are there separate `en_core_news_md` and `en_core_news_lg` models with floret vectors?

I'm not sure if the `en_core_news_md` and `en_core_news_lg` in the notes are a typo, or a separate set of models with floret vectors.

**Comment by polm** [comment, marked as answer]
Sorry this was confusing. The current English pipelines do not use floret vectors, including in 3.4. You can check whether a pipeline uses floret vectors with `nlp.vocab.vectors.mode` - it will be `floret` for floret vectors, or `default` for the standard style vectors that predate floret. 

The blog post does include [some English pipelines with floret vectors](https://explosion.ai/blog/floret-vectors#examples) (and nothing else) for demonstration purposes, but they have names like `en_vectors_floret` - `en_core_news` is just a typo. 

The updates to vectors in 3.4 were to fix long-standing issues like case sensitivity; see #2281.

**Reply by kinghuang** [reply]
Thanks for the clarification!
